db/migrate/20240903123307_change_mentor_training_training_type_default.rb

```diff
@@ -0,0 +1,16 @@
+class ChangeMentorTrainingTrainingTypeDefault < ActiveRecord::Migration[7.1]
+  class MentorTrainingForMigration < ApplicationRecord
+    self.table_name = :mentor_trainings
+  end
+
+  def change
+    change_column_default :mentor_trainings, :training_type, from: nil, to: "initial"
+
+    up_only do
+      # We require callbacks for BigQuery to stay in sync with the data.
+      MentorTrainingForMigration.where(training_type: nil).find_each do |mentor_training|
+        mentor_training.update(training_type: Claims::MentorTraining.training_types[:initial])
```

Comment:
Let's use a hardcoded string here. :)

```suggestion
        mentor_training.update(training_type: "initial")
```